dayrepo/snippets/urls.py

```diff
@@ -11,4 +11,5 @@
     path("employee/", views.get_employee,name="get_employee"),
     path("snippet-list/", views.snippet_list,name="snippet_list"),
     path("snippet-post/", views.snippet_post,name="snippet_post"),
```

Comment:
下の方でコメントしたが、パスにanippetが2回出てくるの違和感あるので以下でいかがか。
```suggestion
    path("post/", views.snippet_post,name="snippet_post"),
```
こうすると、`localhost:xxxx/snippets/post`で出てくるはず